datafusion/optimizer/src/push_down_limit.rs

```diff
@@ -65,7 +65,6 @@ impl OptimizerRule for PushDownLimit {
         };
 
         let Limit { skip, fetch, input } = limit;
-        let input = input;
```

Comment:
I'm not sure why the clippy check in CI didn't detect this but it's showed when I ran clippy in my local.